### ISSUE TITLE
RWD Beta:  Subjective changes and unclear directions

### ISSUE THREAD

**Issue by Kernix13** [freeCodeCamp/freeCodeCamp#44845]
## Affected page

Cafe Menu:
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-3
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-85

Registration Form:
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-forms-by-building-a-registration-form/step-33
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-forms-by-building-a-registration-form/step-38

City Skyline:
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-variables-by-building-a-city-skyline/step-8
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-variables-by-building-a-city-skyline/step-10
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-variables-by-building-a-city-skyline/step-51
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-variables-by-building-a-city-skyline/step-80

## Your code

Not applicable

## Expected behavior

Not applicable

## System

Not applicable

### Cafe Menu:
- [x] **Step 3**: Suggest changing `could be` to `is` :small_blue_diamond: #46063

- [x] **Step 85**: Suggest adding the word(s)  `has` or `has space` after the word `address` :small_blue_diamond: #46071

### Registration Form: 
- [x] **Step 33**: Suggest adding the phrase `which requires a closing tag` (refer to Step 29 as an example) :small_blue_diamond: #46063

- [x] **Step 38**: `margin: 1em auto;` did not center anything as far as I could see. Should the beginning of the sentence be rewritten to something like "Give the `h1` and `p` elements a margin of `1em auto`? :small_blue_diamond: #46071

### City Skyline:
- [x] **Step 8**: Suggest rewording the 1st sentence. I kind of see a border but only after looking for one. I saw the top border grow when I entered the `div` tag. Is the 1st sentence even necessary? :small_blue_diamond: #46071

- [x] **Step 10**: It’s confusing that you need to add HTML *AND* CSS. There is an editor for the HTML, but the instructions do not say that you need to switch to the `styles.css` tab and add your CSS at the bottom. Also, shouldn’t clicking `styles.css` deselect `index.html` and vice versa? Should this step be broken into two steps? I can’t recall another Step in any of the sections where you added an HTML element and a CSS rule. :small_blue_diamond: #46063

- [x] **Step 51**: Confusing. The example is unlike what the expected answer is. Maybe it’s my lack of using linear gradients in this way, but I did not know exactly what I was supposed to do. :small_blue_diamond: #46072

- [x] **Step 80**: Confusing. `...make the left and right borders bigger`. Bigger than what? And why will the side borders be `invisible`? Is that true? I didn't know that. Should an explanation be added? It seems like this lesson needs to be rewritten. :small_blue_diamond: #46063

**Comment by nhcarrigan** [comment]
Thank you for taking the time to share such in-depth feedback. I'm going to open this issue up for discussion, as I think a lot of these have merit and we can find some better approaches.

> Also, shouldn’t clicking styles.css deselect index.html and vice versa?

This is intentional behaviour, to allow campers to have multiple "files" open at a time. It's less necessary in the lessons, but will likely be quite helpful in the certification projects once we've migrated those.

**Comment by moT01** [comment]
I would be fine with all the suggestions here. The first one in the skyline project is probably fine. The rest in that project could maybe use some improvements or clarification. I am going to open it up for contribution in case anyone wants to try and tackle some of them. No need to do them all in one PR.

**Comment by Kernix13** [comment]
I was told that would be the best way for a lot of subjective suggestions, otherwise, it would appear "spammy".

**Comment by moT01** [comment]
What would be the best way? All in one PR?

**Comment by Kernix13** [comment]
I don't know - I'm new to this. That is what I was told to do.

**Comment by moT01** [comment]
There's no real wrong way to do it @Kernix13. It probably would be better to group a few of the changes together. The first four suggestions, for instance, are quite simple and could be grouped together. We didn't really decide on what to do with the rest of them - the one possibly needs to be broken up into two steps which would be a little trickier - we could maybe group all the skyline ones together, as well. Any interest in trying to give any of these a fix @Kernix13? We can help walk you through how to do it.

**Comment by Kernix13** [comment]
If you think it is better, I can create an issue for each of the projects which will just be a copy of my original issue but for each project. When that is done I or you can delete _THIS_ issue. Would that be better?

**Comment by Sboonny** [comment]
I will send PR shortly 

here are the files of the issues, I don't know what to do with.

[Cafe menu step 83](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f45b45d099f3e621fbbb256.md)
[register form step 38](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fad8e6148f310bba7890b1.md)
[skyline step 8](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d0.md)
[skyline step 51](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98fb.md)

**Comment by Kernix13** [comment]
It looks like some of the typos have already been edited in the PR you created. All of these were very picky suggested edits. The 4 you didn't know what to do with can probably be ignored.

**Comment by Sboonny** [comment]
Fair enough, although I think the description in skyline step 53 should change.

I am working on translating it to my native language, and I don't how to translate it, because I don't understand it.

But this can be discussed in freeCodeCamp chat, if you wanna close this.

**Comment by Kernix13** [comment]
So I should click the "_Close issue_"?

**Comment by moT01** [comment]
Why are we closing this? The issue hasn't been resolved. Unless we made new issues for this, I would leave it open.

**Comment by Sboonny** [comment]
I think I have mislead the conversation

>  if you wanna close this

The other files that aren't skyline step 51 , may have been altered by previous  PRs, because I couldn't find the issue to fix. 

Of course, this maybe me lacking in the aspects of understanding the issue deep enough.
 
I will discuss this issue in freeCodeCamp chat in hopes to get better ideas

**Comment by Sboonny** [comment]
@Kernix13 feel free to check the PRs, and if you have something that need to change. please leave a review

**Comment by Sboonny** [comment]
I am wondering, if there is something more I may do?🤔

**Comment by Kernix13** [comment]
> @Kernix13 feel free to check the PRs, and if you have something that want it to change. please leave a review

Sorry Shoomy, I must have missed your reply to me. What PRs? It looks like half of the original issues are marked as completed. Should I fix the remaining ones?

**Comment by Sboonny** [comment]
> Sorry @shoomy, I must have missed your reply to me

No worries🍻

>What PRs? 

After each task there have been pull request, hoping to fix the issue.

> original issues are marked as completed. Should I fix the remaining ones?

If you are okay with the changes, I have implemented and you are okay with how the file looking. 

I am happy to work on it again, if I missed anything

**Comment by Kernix13** [comment]
Got it - I'm okay with the changes - go ahead and close it.
